WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverAdapter.kt

```diff
@@ -75,8 +75,11 @@ class ReaderDiscoverAdapter(
                 is ReaderPostUiState -> {
                     oldItem.postId == (newItem as ReaderPostUiState).postId && oldItem.blogId == newItem.blogId
                 }
+                is ReaderRecommendedBlogsCardUiState -> {
```

Comment:
I had to update this logic to avoid item animation when `isFollowed` property is changed